### ISSUE TITLE
[Reacticket] perms

### ISSUE THREAD

**Issue by XxMr1234GamerzxX** [NeuroAssassin/Toxic-Cogs#42]
ino this is new, but it would be nice if it inherited category permissions

**Comment by NeuroAssassin** [comment]
Yes, I was planning on looking into a way for more permissions to be in there.  I'll leave this open for discussion and tracking.

**Comment by Zenchaeus** [comment]
any update regarding the permissions? especially the read message history. users who creates tickets can't see the message generated by the bot.
